research/format.markdown

```diff
@@ -0,0 +1,138 @@
+
+**\+**   **idn-hostname** : As defined by either RFC 1034 as for hostname, or an internationalized hostname as defined by RFC 5890, section 2.3.2.3 [RFC5890].
+
+**idn-hostname ⊃  hostname**
+
+**\+**   **ipv6** : A string instance is valid against this attribute if it is a valid representation of an IPv6 address as defined in RFC 2373, section 2.2 [RFC2373]. 
+
+An IPv6 address as defined in RFC 4291, section 2.2 [RFC4291].
+
+**\+**   **iri**: A string instance is valid against this attribute if it is a valid IRI, according to [RFC3987].
+
+**\+**   **iri-reference**: A string instance is valid against this attribute if it is a valid IRI Reference (either an IRI or a relative-reference), according to [RFC3987].
+**iri ⊃  uri**
+**iri-reference ⊃  uri-reference**
+
+**\+**   **relative-json-pointer-** : A string instance is valid against this attribute if it is a valid Relative JSON Pointer [relative-json-pointer].
+
+**\+**   **regex** : A regular expression, which SHOULD be valid according to the ECMA 262 [ecma262] regular expression dialect.
+
+
+
+## Draft 7 to 2019-09:
+
+References: [Draft 7](https://json-schema.org/draft-07/draft-handrews-json-schema-validation-01#rfc.section.7), [2019-09](https://json-schema.org/draft/2019-09/draft-handrews-json-schema-validation-02#rfc.section.7)
+
+> Behavior: They SHOULD implement validation, they SHOULD offer an option to disable validation for this keyword → The "format" keyword functions as an annotation, and optionally as an assertion. Also, when treating values as annotations unknown formats SHOULD be collected as annotations.
+
+**\+**    **duration** -  A string instance is valid against this attribute if it is a valid representation according to the "duration" production.
+
+**\+**     **uuid**: A string instance is valid against this attribute if it is a valid string representation of a UUID, according to [RFC4122].
+
+### Changes :
+
+| Keyword   | Previous Description                                                                                                                         | After Description                                                                                                                               |
+|-----------|-----------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| hostname  | As defined by RFC 1034, section 3.1 [RFC1034], including host names produced using the Punycode algorithm specified in RFC 5891, section 4.4 [RFC5891]. | As defined by RFC 1123, section 2.1, including host names produced using the Punycode algorithm specified in RFC 5891, section 4.4 [RFC5891]. |
+
+
+
+
+
+## 2019-09 - 2020-12:
```

Comment:
Before you took an approach of `<from> to <to>` but here you have `-` instead of "to". Please be consistent!